modules/router-store/src/router_store_module.ts

```diff
@@ -125,10 +125,12 @@ export const ROUTER_CONFIG = new InjectionToken(
 export const DEFAULT_ROUTER_FEATURENAME = 'routerReducer';
 
 export function _createDefaultRouterConfig(config: any): StoreRouterConfig {
```

Comment:
Change `config` argument type to `StoreRouterConfig | StoreRouterConfigFunction`

Reply:
I agree